### ISSUE TITLE
Navoigation into and inside Scheduler vertical mode does not work

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#10645]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=31689&sid=386ab9291878e50081a256004c1c40c7)

The current setup we have mirrors the following example, with the exception of we use the mode field in the scheduler config to set it to vertical mode: 

https://bryntum.com/products/calendar/examples/calendar-scheduler/

Keyboard navigation works perfectly per the guide here:

Core/Advanced/A11y

When using horizontal orientation, but when we switch to vertical, we are no longer able to navigate to the events displayed. I have tried modifying the example I linked above to be in vertical alignment by adding:

```
mode : 'vertical',
```

To the timeline config block in app.module.js, and have seen the same issue as well.

Please could you advise on the fix/whether this is actually supported functionality yet?

**Comment by ExtAnimal** [comment]
Dupe of https://github.com/bryntum/support/issues/10592
